enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/converter/MobileTokenConverter.java

```diff
@@ -326,25 +335,37 @@ private static Optional<Attribute> buildAmountConversionAttribute(final Operatio
                 .map(Boolean::parseBoolean)
                 .orElse(false);
 
-        final BigDecimal sourceAmountRaw;
-        final BigDecimal targetAmountRaw;
+        BigDecimal sourceAmountRaw;
+        BigDecimal targetAmountRaw;
+        String sourceAmountFormatted;
+        String targetAmountFormatted;
+        String sourceValueFormatted;
+        String targetValueFormatted;
+        final Locale locale = LocaleContextHolder.getLocale();
+        final String sourceCurrencyRaw = sourceCurrency.get();
+        final String targetCurrencyRaw = targetCurrency.get();
+
         try {
             sourceAmountRaw = new BigDecimal(sourceAmount.get());
             targetAmountRaw = new BigDecimal(targetAmount.get());
```

Comment:
In this case, you need to do it per partes - as when the source amount is not a number but the target amount is Ok, you will put null to the target amount.

Reply:
A nice catch, thanks.